anatomy/tracks/concepts.md

```diff
@@ -41,45 +41,73 @@ concepts
 
 ### File: about.md
 
-**Purpose:** Provide information about the concept for a student who has completed the corresponding concept exercise to learn from and refer back to.
+**Purpose:** Provide more detailed information about the Concept for a student who has completed the corresponding Concept Exercise to learn from and refer back to.
 
 **Presence:** Required
 
-Once the student completes the exercise they will be shown this file, which should provide them with a summary of the concept. If the concept introduces new syntax, syntax samples should be included. At a minimum, this file should contain all information that is introduced in the exercise's [`.docs/introduction.md` document](./concept-exercises#filedocsintroductionmd).
+After completing the corresponding Concept Exercise (otherwise known as "learning" a Concept), the content of the Concept page will change from the introduction.md to the about.md. It should provide them with comprehensive information on what they need to know to be fluent in the concept. At a minimum, this file should contain all information that is introduced in the Concepts' [`introduction.md` document](./concepts.md#file-introductionmd).
```

Comment:
```suggestion
After completing the corresponding Concept Exercise (otherwise known as "learning" a Concept), the Concept page will show the contents of the `about.md` file instead of the `introduction.md` file. The `about.md` file should provide students with comprehensive information on what they need to know to be fluent in the concept. At a minimum, this file should contain all information that is introduced in the Concepts' [`introduction.md` document](./concepts.md#file-introductionmd).
```